migrations/Version20260127132046.php

```
@@ -0,0 +1,29 @@
+{
+    public function getDescription(): string
+    {
+        return 'Removes no longer used page';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('DELETE FROM caf_page WHERE code_page = \'admin-matrice-droits\'');
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)